examples/translation_example.py

```diff
@@ -148,53 +162,231 @@ def parse_arguments():
     return parser.parse_args()
 
 
-def main():
-    args = parse_arguments()
-    print(f"Arguments parsed = {args}")
-    client = get_client(**parse_client_args(args))
-    print(client.dashboard_link)
+TERMINAL_PUNCTUATIONS = (".", "!", "?", ":", ",", ";", ")", "}", "]", '"', "'", "”", "’")
+START_PUNCTUATIONS = ("(", "{", "[", "'", '"', "“", "‘")
```

Comment:
Move this up the code towards the top to keep this well formatted.

Reply:
Done